power_grid_model_c/power_grid_model/include/power_grid_model/component/asym_line.hpp

```diff
@@ -16,7 +14,9 @@
 #include "../common/common.hpp"
 #include "../common/matrix_utils.hpp"
 #include "../common/three_phase_tensor.hpp"
-#include "line_utils.hpp"
+
+#include <iostream>
```

Comment:
@mgovers can you explain why do we need `iostream` here?

Reply:
i missed this. this was auto-formatted. will remove in a separate PR